tests/importer.rs

```diff
@@ -0,0 +1,252 @@
+use std::collections::{HashMap, HashSet};
+use std::fs;
+
+use hurban_selector::importer;
+use hurban_selector::importer::{Importer, ImporterError};
+
+fn import_obj(path: &str) -> Vec<importer::Model> {
+    let file_contents = fs::read_to_string(&path).expect("File should be read to string");
+    let (tobj_models, _) =
+        importer::obj_buf_into_tobj(&file_contents).expect("Obj should be parsed");
+
+    importer::tobj_to_internal(tobj_models)
+}
+
+#[test]
+fn test_caches_valid_obj_file() {

```

Comment:
How about having a `has_cached(&str)` on `Importer`? You wouldn't need to have all those items public and would still be able to test exactly what you want to:

```
create importer;
use importer to load model;
ensure model is cached;
```

Reply:
Yes, it's a possibility, I'll look into it.